core/src/main/scala/persistentscheduler/PersistentScheduler.scala

```diff
@@ -111,15 +116,15 @@ class PersistentScheduler(persistence: SchedulerPersistence) extends Actor
       nextCancellable = nextEvent.map(schedule(_))
     }
 
-    sender() ! Scheduled(event)
+    sender() ? Scheduled(event)
   }
 
   def schedule(event: TimedEvent): Cancellable = {
     val after = Math.max(event.date.getMillis - DateTime.now().getMillis, 0L).millis
     scheduler.scheduleOnce(after, self, PublishEvent(event))
```

Comment:
i guess we should also wrap this into a Try. this was the problem we had 1 or 2 weeks ago, where it threw an exception here if the date was too far in the future